README.md

```diff
@@ -41,10 +41,28 @@ in Internet Explorer on Windows 7 for instance (because of the wrong MIME type).
 
 JavaScript Cookie supports [npm](https://www.npmjs.com/package/js-cookie) and [Bower](http://bower.io/search/?q=js-cookie) under the name `js-cookie`.
 
+- #### Bower
+  ```
+  bower install js-cookie --save
```

Comment:
It's not about teaching them their own tools, it's a convenience thing. I love it when it's done for me cause then I copy and paste the command knowing it'll work. 

It's a mixture between commonplace and specificity.